core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -244,7 +249,8 @@ class KafkaApis(val requestChannel: RequestChannel,
               s"on partition $topicPartition failed due to ${Errors.forCode(errorCode).exceptionName}")
```

Comment:
Thanks for updating the code so that these errors are logged as per my comment above. I thought that we should indicate in the server log that the issue is lack of describe instead of a missing topic. Something as simple as some extra text at the end: "as the user u does not have DESCRIBE permission" (something like that anyway).